readme/publish_note.md

```diff
@@ -13,3 +13,11 @@ After that a dialog opens showing you the note - from there you just need to cli
 You can then share this link with anyone and they will be able to see the note in their browser:
 
 <img src="https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/publish_note/Browser.jpg" width="75%"/>
+
+# Unpublishing a note
+
+After publishing a note, it can be unpublished by right-clicking on it in the Joplin desktop application, and selecting "Publish note..."
+
+<img src="https://raw.githubusercontent.com/MattDemers/joplin/dev/Assets/Screenshots/joplin_unpublish_note.png" width="75%">
+
+From there, clicking on the "Share" icon will remove the icon, and unpublish the note.
```

Comment:
"From there, click on the "Share" icon to unpublish the note."

Reply:
Is the "Share" icon so universal that people would know what to look for?

Reply:
Also I'd like to make it clear that the icon will be removed/disappear, so the user has some feedback that their change did something.

Reply:
If it's not known then we need to use a different icon, or add text. It needs to be usable as it is without having to read the documentation.

Also what the user wants to do here is unpublish the note. Again, we can hope that when the icon disappears the user understands that it means the note is no longer published.

Basically I'm fine adding a section about this but it should be concise, with the assumption that we got the UX more or less right. If not, it's a bigger issue that the doc won't fix.

Reply:
Alright, makes sense. In my experience, the icon that you've chosen to use is more for *getting* a link to share, or to share to other apps. Then again, that isn't the purpose of this PR (thanks for the reminder).